solutions/cloud_scaling_overview/index.asciidoc

```diff
@@ -0,0 +1,80 @@
+//Category=Scaling
+//Product=Scaling
+//Maturity level=Initial
+
+= Scalability in Cloud
+
+== Scalability in cloud
+
+Capacity planning is a challenge every engineering team faces when it comes to ensuring the right resources are in place to handle expected and unexpected traffic demands. You might end up investing too much for your infrastructure or it will be underrun which will affect your application performance. To avoid such problems, we should have scalable application. Cloud scalability makes it easy to handle this changing needs.  
+
+=== What is scalability?
+
+Cloud scalability in cloud computing refers to the ability to increase or decrease IT resources as needed to meet changing demand. Data storage capacity, processing power and networking can all be scaled using existing cloud computing infrastructure. Scalability is one of the biggest advantage of using cloud.
+
+Scalable application can handle changing workload in pre-planned manner.
+
+=== Benefits of cloud computing:
+
+* Convenience:
+It was taking weeks to months to scale an application on premise environment but with cloud it has become easy. We can add more resources on few clicks.
+* Cost saving:
+This helped users to reduce capex and changed it to opex. 
+* Better user experience:
+Scalable application gives better user experience and have increased performance. 
+* Disaster recovery: 
+With scalable cloud computing, you can reduce disaster recovery costs by eliminating the need for building and maintaining secondary data centres.
+
+=== Types of scalability:
+There are two types of scalability:
+
+*	Vertical scaling
+*	Horizontal scaling
+
```

Comment:
Can we portray the differences b/w vertical and horizontal scaling in a tabular format? that way it makes it more readable.